roles/cluster_issuer/tasks/type/none/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
can you please add the usual apache2 license header

Reply:
fixed